packages/app/src/views/AddModule/wizards/ConnextModule/ConnextModuleModal.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react"
+import { Typography, makeStyles } from "@material-ui/core"
+import { ParamType } from "@ethersproject/abi"
+import { ConnextModuleParams, deployConnextModule } from "services"
+import { ReactComponent as ArrowUpIcon } from "../../../../assets/icons/arrow-up-icon.svg"
+import useSafeAppsSDKWithProvider from "hooks/useSafeAppsSDKWithProvider"
+import { ParamInput } from "../../../../components/ethereum/ParamInput"
+import { ActionButton } from "../../../../components/ActionButton"
+import { AddModuleModal } from "../components/AddModuleModal"
+
+interface ConnextModuleProps {
+  open: boolean
+  onClose?(): void
+  onSubmit?(): void
+}
+
+const useStyles = makeStyles((theme) => ({
+  addButton: {
+    marginTop: theme.spacing(2),
+  },
+  addTransactionButton: {
+    marginTop: theme.spacing(1),
+  },
+  addIcon: {
+    stroke: theme.palette.common.white,
+    width: 20,
+    height: 20,
+  },
+  inputParam: {
+    marginTop: theme.spacing(2),
+  },
+}))
+
+export const ConnextModuleModal = ({ onSubmit, open, onClose }: ConnextModuleProps) => {
+  const { sdk, safe, provider } = useSafeAppsSDKWithProvider()
+  const classes = useStyles()
+
+  const [params, setParams] = useState<ConnextModuleParams>({
+    owner: safe.safeAddress,
+    avatar: safe.safeAddress,
+    target: safe.safeAddress,
+    domainId: 0,
+    sender: "",
+  })
+
+  const onParamChange = <Field extends keyof ConnextModuleParams>(
+    field: Field,
+    value: ConnextModuleParams[Field],
+    valid?: boolean,
+  ) => {
+    setParams({
+      ...params,
+      [field]: value,
+    })
+  }
+
+  const handleAddConnextModule = async () => {
+    try {
+      const args = {
+        ...params,
+      }
+      const txs = deployConnextModule(provider, safe.safeAddress, safe.chainId, args)
+
+      await sdk.txs.send({ txs })
+      if (onSubmit) onSubmit()
+      if (onClose) onClose()
+    } catch (error) {
+      console.log("Error deploying module: ", error)
```

Comment:
console.warn or error would be better in this case

Reply:
https://github.com/defi-wonderland/zodiac-safe-app/pull/1/commits/1b1231b0bfd7a8686faad53fa8e9f153b0e1de24